dev-docs/bidders/lm_kiviads.md

```diff
@@ -2,24 +2,33 @@
 layout: bidder
 title: LM KiviAds
 description: LM Kiviads Bidder Adapter
-biddercode: xe
-media_types: banner, video, native
+biddercode: lm_kiviads
+media_types: banner, video
 coppa_supported: true
 tcfeu_supported: false
 usp_supported: true
 prebid_member: false
-pbjs: false
+pbjs: true
 pbs: true
 schain_supported: true
 floors_supported: true
 multiformat_supported: will-bid-on-any
 sidebarType: 1
 ---
 
+### Prebid.js Bid params
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                 | Example       | Type      |
+|-------------|----------|-----------------------------|---------------|-----------|
+| `placement` | required | Placement ID                | `test-banner` | `string`  |
+| `env`       | required | Environment name            | `lm_kiviads`  | `string`  |
+| `ext`       | optional | Specific integration config | `{}`          | `object`  |
+
 ### Prebid Server Bid Params
 
 {: .table .table-bordered .table-striped }
 | Name        | Scope    | Description                 | Example                            | Type      |
 |-------------|----------|-----------------------------|------------------------------------|-----------|
-| `env`       | required | Environment name            | `kivi-stage`                       | `string`  |
-| `pid`       | required | unique placement ID           | `cs230510321b516f0eb9a10e5913d3b1` | `string`  |
+| `env`       | required | Environment name            | `lm_kiviads-stage`                 | `string`  |
+| `pid`       | required | Unique placement ID           | `cs230510321b516f0eb9a10e5913d3b1` | `string`  |
```

Comment:
Publishers are unhappy when client-side parameters differ from server-side parameters because it makes their lives more difficult. I've added this adapter to the list at https://github.com/prebid/prebid-server/issues/2654 -- please resolve ASAP.